src/dependencyReportPanel.ts

```diff
@@ -158,5 +163,26 @@ let render_project_failure = () => {
 
 let render_stack_iframe = portaluri => {
   //const result = sa.result[0];
-  return `<iframe width="100%" height="100%" frameborder="0" src=${portaluri} id="frame2" name="frame2"></iframe>`;
+  return `<iframe id="frame" width="100%" height="100%" frameborder="0" src=${portaluri} sandbox="allow-scripts allow-popups allow-popups-to-escape-sandbox allow-same-origin"></iframe>
+  
+  <script>
+
+  var eventMethod = window.addEventListener ? "addEventListener" : "attachEvent";
```

Comment:
VSCode WebView runs on Electron based browser runtime, so you can assume `window. addEventListener` will be there always. This would eliminate the whole bunch of code related to this.

Reply:
Okay..